packages/nextjs/services/onchainData/ens.ts

```diff
@@ -0,0 +1,27 @@
+import { publicClient } from "~~/utils/short-address-and-ens";
+
+export type EnsData = {
+  name: string | null;
+  avatar: string | null;
+};
+
+export async function fetchEnsData(address: string): Promise<EnsData> {
```

Comment:
naming: we are still using `get` and `fetch`. Not sure if we agree in something, but I think `fetch` makes sense when we are making an external request (and get for database calls, etc)

Reply:
Makes sense! And we are kind of also following it 🙌 maybe we mentioned this in one of the meet